addons/hr_holidays/tests/test_leave_requests.py

```diff
@@ -1563,3 +1563,13 @@ def test_time_off_creation_without_allocation(self):
                 'request_date_from': '2024-07-01',
                 'request_date_to': '2024-07-02',
             })
+
+    def test_request_date_to_doesnt_raise_error(self):
```

Comment:
```suggestion
    def test_duration_display_without_request_date_to(self):
```
Name should relative